slack_api/sender.py

```diff
@@ -73,6 +73,11 @@ def restaurant_meal_blocks(restaurant: Restaurant, meals: list, user_meals_pks:
                  ] + [
                      SlackSender._meal_voting_block(m, (user_meals_pks is not None) and (m.pk not in user_meals_pks)) for m in meals
                  ]
+        if len(blocks) > 50:
```

Comment:
This case actually seems like an error (as it was with Enterprise). Maybe we should fail fast on this?

Reply:
Now showing an error message via Slack instead, is that better? Throwing an exception is not that visible...